.config/openbox/menu.xml

```diff
@@ -333,7 +333,7 @@
             <item label="User Login Settings">
                 <action name="Execute">
                     <command>
-                        gksudo geany /etc/lightdm/lightdm-gtk-greeter.conf /etc/lightdm/lightdm.conf
+                        gksudo bl-text-editor /etc/lightdm/lightdm-gtk-greeter.conf /etc/lightdm/lightdm.conf
```

Comment:
Just a thought - we are assuming here that bl-text-editor, like geany and gedit, will take multiple arguments. Leaf/Mousepad, for example, don't. Maybe it would be safer to have two separate \<action tags, one for each file?

    <action name="Execute">
        <command>gksudo bl-text-editor /etc/lightdm/lightdm-gtk-greeter.conf</command>
    </action>
    <action name="Execute">
        <command>gksudo bl-text-editor /etc/lightdm/lightdm.conf</command>
    </action>